libs/sql-schema-describer/src/mysql.rs

```diff
@@ -578,17 +587,33 @@ fn get_column_type_and_enum(
     full_data_type: &str,
     precision: Precision,
     arity: ColumnArity,
+    default: Option<&Value>,
 ) -> (ColumnType, Option<Enum>) {
+    // println!("Name: {}", column_name);
     // println!("DT: {}", data_type);
     // println!("FDT: {}", full_data_type);
     // println!("Precision: {:?}", precision);
+    // println!("Default: {:?}", default);
+
+    let is_tinyint1 = || extract_precision(full_data_type) == Some(1);
+    let invalid_bool_default = || match default {
+        None => false,
+        Some(param_value) => match param_value.to_string() {
```

Comment:
nitpick: I'm not sure the `to_string()` is needed here (maybe `match &param_value`)

Reply:
The param_value is still a quaint::Value at that point

Reply:
Ah indeed, that `to_string()` is always super confusing to me, because it overlaps with the `Display` trait.